src/legacy/core_plugins/timelion/public/directives/_index.scss

```diff
@@ -1,6 +1,7 @@
 @import './timelion_expression_input';
 @import './cells/index';
-@import './chart/index';
 @import './timelion_expression_suggestions/index';
 @import './timelion_help/index';
 @import './timelion_interval/index';
+
+@import '../../../vis_type_timelion/public/components/chart'
```

Comment:
Why does this import need to be duplicated here? It's already imported via `vis_type_timelion/public/index.scss`.

Reply:
I thought if a plugin uses styles from other plugin, it should import directly because there is no guarantees that other plugin will be loaded.
Please let me know if I'm wrong.

Reply:
That might be correct **eventually**. But at the moment all the CSS is loaded on every page so this import will duplicate the styles coming from this file. I would hope that once migration is over, the NP will be able to setup imports of CSS alongside the components that need them.

Reply:
Thanks for explanation.
So in that case I removed this import (anyway the Timelion app will be removed soon)